services/worker/tests/fixtures/descriptive_statistics_dataset.py

```diff
@@ -862,7 +862,553 @@ def nan_column() -> list[None]:
+            [],
+            [],
+            [],
+            [],
+        ],
+        "list__sequence_class_label_nan_column": [
+            ["cat"],
+            ["cat"],
+            None,
+            ["cat", "dog"],
+            ["cat", "cat"],
+            ["cat", "cat"],
+            ["cat", "cat"],
+            None,
+            ["cat", "cat", "cat", "dog"],
+            ["cat", "cat", "cat", "dog"],
+            ["cat", "cat", "cat", "dog", "cat", "cat", "dog"],
+            ["cat", "cat", "cat", "dog", "cat", "cat", "dog"],
+            None,
+            None,
+            ["cat", "cat", "cat", "dog", "cat", "cat", "dog"],
+            ["cat", "cat", "cat", "dog", "cat", "cat", "dog"],
+            [],
+            [],
+            [],
+            [],
+        ],
+        "list__sequence_class_label_all_nan_column": nan_column(),
+        "list__sequence_of_sequence_bool_column": [
+            [[True]],
+            [[True]],
+            [[True]],
+            [[True], [None, True, False], []],
+            [[True, False]],
+            [[True, False], [True, False], [True]],
+            [[True]],
+            [[True, False, False], [True, False], [], [True, False], [True, False, True, False]],
+            [[True, False, False]],
+            [[True, False, False]],
+            [[True, False, False]],
+            [[True, False, False, False], [True], [], [None, True]],
+            [[True, False, False, False], [True], [None, True]],
+            [[True, False, False, False, True], [None, True]],
+            [[True], [False], [False], [False, None], [True, None, True]],
+            [[True]],
+            [[True, False], [False, False, True], [None, True]],
+            [[True, False, False, False], [None, True]],
+            [[True], [None, True, False]],
+            [[True], [True, False], [True, False]],
+        ],
+        "list__sequence_of_sequence_bool_nan_column": [
+            [[True]],
+            [[True]],
+            [[True]],
+            [[True], [None, True, False]],
+            None,
+            [[True, False], [True, False], [True], []],
+            [[True]],
+            [[True, False, False], [True, False], [True, False], [], [True, False, True, False]],
+            [[True, False, False]],
+            None,
+            [[True, False, False]],
+            [[True, False, False, False], [True], [None, True]],
+            [[True, False, False, False], [True], [None, True]],
+            [[True, False, False, False, True], [None, True]],
+            [[True], [False], [False], [False, None], [True, None, True]],
+            [[True]],
+            None,
+            [[True, False, False, False], [None, True]],
+            [[True], [None, True, False]],
+            [[True], [True, False], [True, False]],
+        ],
+        "list__sequence_of_sequence_bool_all_nan_column": nan_column(),
+        "list__sequence_of_sequence_dict_column": [
```

Comment:
this case would work but seems to be always of length=1 because internally it's 
```
[
	[{'author': ['cat'], 'likes': [5]}], 
	[{'author': ['cat', 'cat'], 'likes': [5, 5]}], 
	[{'author': ['cat', 'cat'], 'likes': [5, 5]}], 
         ...
 	[{'author': [], 'likes': []}]
]
```
i don't expect to see this in reality though :D